.gitignore

```diff
@@ -15,4 +15,7 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+# Mac OS files
 .DS_Store
+.DS_Store?
```

Comment:
I'm not sure this line is necessary. I'm particularly intrigued by the question mark.

Of course, if it's needed then just reply to this and I'm happy to approve

Reply:
https://stackoverflow.com/questions/107701/how-can-i-remove-ds-store-files-from-a-git-repository

Reply:
Hm so there's a different file called .DS_Store? or it's just some git magic. Thanks, I've learned something today...and come up with a lot more questions to Google.

Anyway, LGTM

Reply:
http://osxdaily.com/2009/12/31/what-is-a-ds_store-file/

Reply:
I've learned a lot today!

Reply:
Btw, I know about the .DS_Store file. I was wondering why we also have one called ".DS_Store"

Reply:
It probably was added by myself in the repository by mistake. Once it happens it seems to be generated automatically. The .DS_Store? is a hidden file... so I had no clue what was going on. So then Katja helped me and we could find the file by using the command + shift + .      So then I could delete it.

Reply:
There was this really weird thing happening with that file that just wouldn't stop being included in git even though we have added it to .gitignore. This might be a solution but we will have to see. It kept popping up on Shirley's mac yesterday.

Reply:
You might have to clear the git cache before the new git ignore. On the command line you use `git rm -r . --cached` and I'm sure there's something similar for git GUIs.

If it's fixed now then that's good. Otherwise, you can use the cached thing. Deleting the file from GitHub and then pulling might also fix it

Reply:
It seems that it is fixed now. But if pop out again I do it